ceph.spec.in

```diff
@@ -1183,6 +1186,9 @@ fi
 %{_mandir}/man8/ceph-clsinfo.8*
 %{_mandir}/man8/ceph-disk.8*
 %{_mandir}/man8/ceph-osd.8*
+%if 0%{?rhel}
+/etc/cron.hourly/subman.py
```

Comment:
I'm not sure how would cron handle the additional files in the directory. Also, we do not generally ship these files for python's executable scripts, e.g. those that get installed to /usr/bin/ -- even python's setuptools do not create them for these.